code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1535,6 +1540,8 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	var/Iforce = I.force //to avoid runtimes on the forcesay checks at the bottom. Some items might delete themselves if you drop them. (stunning yourself, ninja swords)
 
 	var/weakness = H.check_weakness(I, user)
+	if (I.damtype == STAMINA && H.nutrition < NUTRITION_LEVEL_STARVING)
+		I.force*=2 //stamina fat multiplier under NUTRITION_LEVEL_STARVING
```

Comment:
Double stamina damage is pretty crazy, I would go for values like 1.2 for starving and 1.4 for super starving or whatever.
Also why would being hungry make you take more stamina damage?

Reply:
Well I thought of it as an incentive for people to eat, people wouldn't stay hungry for the buff, and muscles have less protein and energy so they tire faster from stamina.